packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -19,10 +20,10 @@ import {
     SequenceChangeset,
     SequenceEditBuilder,
 } from "../feature-libraries";
-import { IEditableForest, IForestSubscription } from "../forest";
+import { IForestSubscription } from "../forest";
 import { StoredSchemaRepository } from "../schema-stored";
 import { Index, SharedTreeCore } from "../shared-tree-core";
-import { Checkout, runSynchronousTransaction, TransactionResult } from "../transaction";
+import { Checkout as TransactionCheckout, runSynchronousTransaction } from "../transaction";
```

Comment:
It doesn't look like `Checkout` would conflict with anything. Is this alias intentional, or an artifact of a previous state of the code?

Reply:
This was intentional, though it turns out there isn't a name collision because this this checkout does not use the `I` Prefix like the other one does. We should fix then, and then they will collide.